### DISCUSSION TITLE
Creating a Website

### DISCUSSION THREAD

**Discussion by Soontosh** [SpotX-Official/SpotX#472] [Ideas]
Hey!

I've got quite a bit of experience with full-stack development. I was wondering if I would be allowed to convert the installer to a .exe file, create, and host a website for SpotX? I'm thinking of just making a landing page with information, animations, and a download button. I can handle databases and add a backend with Python and SQL, although I don't know if that will be necessary for this project.

I just wanted to ask for permission before I made the website, and to ask if this has already been created.

**Comment by amd64fox** [comment]
Hello, if you are ready to help free of charge, In that case, I invite you to join our small team. Any help is valuable to us.

We need a documentation website, and it's not necessary to build it from scratch. We can use any existing platform like Docsify. The specifics can be discussed.

Regarding the installer, I'm not entirely clear on what you mean. Are you looking to rewrite the patcher itself, or are you referring to installing an already patched Spotify client?

**Reply by Soontosh** [reply]
Nevermind about rewriting the patcher, I realize now that it is unnecessary. I'm busy till Monday, but after that I can get to work. I have a few questions.

Is there anything specific about the documentation I need to know? 
Is there a way you would want me to organize the docs? 
Also, how will I know what to include? Are the docs already available in a text format?
If you would like, I can help with making a website outside of documentation, like maybe a download page, a subscription, or anything you feel would make the website better.

I am excited to start helping!

**Reply by amd64fox** [reply]
go here [t.me/amd64fox](https://t.me/amd64fox)

**Reply by Soontosh** [reply]
I'll get back to you as soon as I can. I'm busy till Monday.

**Reply by EdwardDK** [reply]
@amd64fox I also would want to start helping on the website. I alsojust do front-end dev.

Discord: .eddy47

**Reply by amd64fox** [reply]
Guys, you can either write to me using the link above, or you can start working on the website right away, I don't mind, fewer empty talks, more action. 

Cheers

**Reply by EdwardDK** [reply]
I don't have telegram

**Reply by amd64fox** [reply]
@Soontosh 
@EdwardDK 
I sent an invitation to a new repo for the site, try to start doing something together if you still have a desire

**Reply by Soontosh** [reply]
Okay, I'll check it out. Edward, could you please send your Discord so we can communicate? I'm thinking of using a simple stack, with HTML, CSS, and Javascript. If needed, I could use Flask or FireBase for backend.

**Reply by EdwardDK** [reply]
.eddy47

**Reply by Soontosh** [reply]
I send you a friend request. Please accept so we can get started!

**Comment by EdwardDK** [comment]
Ok, this didn't go anywhere. @Soontosh you have to reply

**Reply by shnjd** [reply]
Hello @EdwardDK I'm interested to join this. Can I contact you thorugh telegram?

**Reply by shnjd** [reply]
@amd64fox Can I join this work? I don't think we need a backend for this. We could just make a documentation/guide website with a landing page. I'm an experienced frontend dev, you can go through my gh profile.

**Reply by EdwardDK** [reply]
I don't have telegram, maybe there are some other ways you can communicate with me?

**Reply by shnjd** [reply]
@EdwardDK , I've started working on this last week. Maybe I can communicate with you using discord, but we already created a group in telegram..
